### ISSUE TITLE
go get: cannot find package "golang.org/x/tools/cmd/vet"

### ISSUE THREAD

**Issue by wolfgang42** [drone/drone#1565]
[This error](http://beta.drone.io/drone/drone/354) happens whenever building drone:
```
$ make deps gen
go get -u golang.org/x/tools/cmd/cover
go get -u golang.org/x/tools/cmd/vet
package golang.org/x/tools/cmd/vet: cannot find package "golang.org/x/tools/cmd/vet" in any of:
	/usr/local/go/src/golang.org/x/tools/cmd/vet (from $GOROOT)
	/drone/src/golang.org/x/tools/cmd/vet (from $GOPATH)
Makefile:8: recipe for target 'deps' failed
make: *** [deps] Error 1
[info] build failed (exit code 2)
```

[A discussion on the drone gitter channel](https://gitter.im/drone/drone-dev?at=5706a3db1ee04bd23b764aa7) with @bradrydzewski and @tboerger determined that [go has deprecated x/tools](https://groups.google.com/forum/#!topic/golang-announce/qu_rAphYdxY). Removing this line from the Makefile entirely seems to fix the problem:
```diff
diff --git a/Makefile b/Makefile
index ca2fc29..632c8bf 100644
--- a/Makefile
+++ b/Makefile
@@ -6,7 +6,6 @@ all: gen build

 deps:
        go get -u golang.org/x/tools/cmd/cover
-       go get -u golang.org/x/tools/cmd/vet
        go get -u github.com/eknkc/amber/...
```

Drone *seems* to work when you do this; I don't know enough about go to be certain that this won't break anything.

**Comment by mandeepbal** [comment]
+1 - I'm seeing the same issue

**Comment by bradrydzewski** [comment]
closed by #1567
